integration-tests/src/test/java/org/apache/druid/tests/parallelized/ITSqlInputSourceTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.tests.parallelized;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.druid.indexer.partitions.DynamicPartitionsSpec;
+import org.apache.druid.java.util.common.StringUtils;
+import org.apache.druid.testing.guice.DruidTestModuleFactory;
+import org.apache.druid.tests.TestNGGroup;
+import org.apache.druid.tests.indexer.AbstractITBatchIndexTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Guice;
+import org.testng.annotations.Test;
+
+import java.io.Closeable;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Function;
+
+@Test(groups = TestNGGroup.BATCH_INDEX)
+@Guice(moduleFactory = DruidTestModuleFactory.class)
+public class ITSqlInputSourceTest extends AbstractITBatchIndexTest
+{
+  private static final String INDEX_TASK = "/indexer/wikipedia_index_task.json";
+  private static final String INDEX_QUERIES_RESOURCE = "/indexer/wikipedia_index_queries.json";
+
+  @DataProvider(parallel = true)
+  public static Object[][] resources()
+  {
+    return new Object[][]{
+        // Multiple query. No filter
```

Comment:
nit: Maybe consider making the name of the test part of the parameters, then you can use the `@TestCaseName` annotation to print out the name of each parameterized test and it could also be used as the name of the datasource.

like `@TestCaseName("testSqlInputSource - {1}")`

Reply:
This is not needed. 
In travis log we already have a LoggerListener that reports when test starts and the status (pass, fail, etc) of the test. The LoggerListener also report out the parameter.
For example,
`2020-06-26T02:33:00,617 INFO [TestNG-PoolService-0] org.apache.druid.testing.utils.LoggerListener - Passed [org.apache.druid.tests.parallelized.ITSqlInputSourceTest.testIndexData] with parameters [[SELECT * FROM wikipedia_index_data1 WHERE added > 0, SELECT * FROM wikipedia_index_data2 WHERE added > 0, SELECT * FROM wikipedia_index_data3 WHERE added > 0]]``
2020-06-26T02:33:00,617 INFO [TestNG-PoolService-0] org.apache.druid.testing.utils.LoggerListener - Starting [org.apache.druid.tests.parallelized.ITSqlInputSourceTest.testIndexData] with parameters [[SELECT * FROM wikipedia_index_data_all WHERE timestamp BETWEEN '2013-08-31 00:00:00' AND '2013-09-01 11:59:59']]`